nengo/spa/assoc_mem.py

```diff
@@ -60,8 +54,9 @@ def __init__(self, input_vocab, output_vocab=None,  # noqa: C901
                  default_output_key=None, threshold=0.3,
                  inhibitable=False, wta_output=False,
                  wta_inhibit_scale=3.0, wta_synapse=0.005,
-                 threshold_output=False, label=None, seed=None,
-                 add_to_container=None):
+                 cleanup_output=False,
+                 replace_output_with_cleaned_output=True,
+                 label=None, seed=None, add_to_container=None):
```

Comment:
Unless you're going to have docstrings for `label`, `seed`, and `add_to_container`, I would maybe do them as `**network_kwargs` and just note in the docstring that they go straight to the Network constructor so users can look them up.